sfepy/examples/linear_elasticity/modal_analysis_declarative.py

```diff
@@ -107,6 +107,10 @@ def define(n_eigs=6, approx_order=1, density=7850., young=210e9, poisson=0.3):
             'which': 'LM',
             'sigma': 0,
             'tol': 1e-8,
+            'linear_solver': ('ls.mumps', {}),
```

Comment:
I would set the solver to something that is not optional dependency. MUMPS libraries need not be available.

Reply:
The default linear solver is set to `ls.scipy_superlu` and the comment is updated.